packages/roosterjs-editor-api/lib/format/toggleBullet.ts

```diff
@@ -1,9 +1,36 @@
 import execFormatWithUndo from './execFormatWithUndo';
-import { Editor } from 'roosterjs-editor-core';
+import getNodeAtCursor from '../cursor/getNodeAtCursor';
+import { Editor, browserData } from 'roosterjs-editor-core';
+import { NodeType } from 'roosterjs-editor-types';
+
+// Edge may incorrectly put cursor after toggle bullet, workaround it by adding a space.
+// The space will be removed by Edge after toggle bullet
+export function workaroundForEdge(editor: Editor): boolean {
+    if (browserData.isEdge) {
+        let node = getNodeAtCursor(editor);
+        if (node.nodeType == NodeType.Element && (<Element>node).textContent == '') {
```

Comment:
If it's an image element, would textContent also be ''? seems to be okay though, just curious.

Reply:
Good cache. I have made a fix for this scenario.